package.json

```diff
@@ -20,7 +20,7 @@
 		"@wordpress/dom-ready": "1.0.4",
 		"@wordpress/hooks": "1.1.6",
 		"@wordpress/i18n": "1.1.0",
-		"@wordpress/is-shallow-equal": "1.0.1",
```

Comment:
Any particular reason for this upgrade?

Reply:
Yes, there were some issues with the package outside of Gutenberg and it wasn't bumped in Gutenberg. See: https://github.com/WordPress/packages/pull/124.
We can do it separately.

Reply:
No, I'm fine keeping it, just curious